modules/backend/controllers/Auth.php

```diff
@@ -167,32 +168,28 @@ public function restore_onSubmit()
         ];
 
         $validation = Validator::make(post(), $rules);
-        if ($validation->fails()) {
-            throw new ValidationException($validation);
-        }
```

Comment:
The validation failing should still come into play here shouldn't it? I don't see how that's leaking any information

Reply:
@LukeTowers A different statuscode can also leak information.

Reply:
Doesn't the ValidationException return a 406 http error code? Really any indication that could give away that an account exists should be avoided.

Reply:
@LukeTowers like @jacobdekeizer and @Klaasie mentioned, I consider it's better to not display the information that a username must have a specific length.

Reply:
@LukeTowers like @jacobdekeizer and @Klaasie mentioned, I consider it's better to not display the information that a username must have a specific length.

Reply:
@jacobdekeizer @cosminbosutar @Klaasie the validation exception has nothing to do with whether an account exists or not, it just an early indicator to the user that they screwed up their request. My main problem with it right now is that it's not necessarily accurate given some DB configurations result in a max length of string columns of 191; so on that basis I'm fine with removing the length part, but validating that the input is required period is probably still a good idea.

Reply:
@cosminbosutar any thoughts on the above?

Reply:
@LukeTowers I agree with you that validating that the input is received is a good idea.
I'll make an update to the PR in which I'll add the logic that validates that the input exists and the validation messages.

Reply:
@cosminbosutar just waiting for your update.